sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableServiceClientTest.java

```diff
@@ -351,10 +351,8 @@ public void generateAccountSasTokenWithAllParameters() {
     }
 
     @Test
-    // Disabling as this currently fails and prevents merging https://github.com/Azure/azure-sdk-for-java/pull/28522.
-    // TODO: Will fix in a separate PR. -vicolina
     public void canUseSasTokenToCreateValidTableClient() {
-        final OffsetDateTime expiryTime = OffsetDateTime.of(2023, 12, 12, 0, 0, 0, 0, ZoneOffset.UTC);
+        final OffsetDateTime expiryTime = OffsetDateTime.now().plusDays(1);
```

Comment:
This should use `TestResourceNamer` to get a now datetime, I believe this may break with the current design

Reply:
Let's punt this feedback to another PR as there are some other things going on here that already existed and not part of this PR